src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -291,16 +312,16 @@ interface LayoutProps
     | "order"
     | "me"
     | "setFulfillmentFormHelpers"
-    | "onSubmit"
     | "active"
     | "renderMissingShippingQuotesError"
+    | "onSubmit"
   > {
   verifyAddressNow: boolean
-  addressVerificationComplete: () => void
+  addressVerificationComplete: () => Promise<void>
 }
 
 const FulfillmentDetailsFormLayout = (props: LayoutProps) => {
-  const { active } = props
+  const { active, onSubmit: submitWithoutFormik } = props
```

Comment:
The way this branches between the formik submit handler and here is a bit confusing